docs/status.md

```diff
@@ -90,8 +90,8 @@ one of the following tracking labels.
 | log_plus_one             |      no       |     yes*     |       yes*        |        yes        |     no      |
 | logistic                 |      yes      |     yes*     |       yes*        |        yes        |     no      |
 | map                      |      no       |     yes*     |        no         |        no         |     no      |
-| maximum                  |      yes      |     yes*     |       yes*        |        yes        |     no      |
-| minimum                  |      yes      |     yes*     |       yes*        |        yes        |     no      |
+| maximum                  |      yes      |     yes      |       yes         |        yes        |     yes     |
```

Comment:
Should both ops stay as `yes*` due to #262?

Reply:
Nvm, I didn't see the nearby PR. Linking for reference #264.

Reply:
Good catch! Yes, it should back at "yes*"

Reply:
Yes, I will switch it back once that PR is merged.